packages/manager/modules/vps/src/dashboard/tile/configuration/index.html

```diff
@@ -74,7 +74,10 @@
             <p
                 class="mt-2 mb-0"
                 data-translate="vps_dashboard_tile_configuration_memory_double"
-                data-translate-values="{ price: $ctrl.configurationTile.isUpfront ? $ctrl.configurationTile.upgrades.memory.upfrontDiff.text : $ctrl.configurationTile.upgrades.memory.diff.text }"
+                data-translate-values="{
+                    additionalPrice: $ctrl.configurationTile.isUpfront ? $ctrl.configurationTile.upgrades.memory.upfrontDiff.text : $ctrl.configurationTile.upgrades.memory.diff.text,
+                    price: $ctrl.configurationTile.isUpfront ? $ctrl.configurationTile.upgrades.memory.upfrontTotal.text : $ctrl.configurationTile.upgrades.memory.total.text
+                }"
```

Comment:
Can we move the price calculation part to controller for better readability ? 
```suggestion
                data-translate-values="{
                    additionalPrice: $ctrl.configurationTile.isUpfront ? $ctrl.configurationTile.upgrades.memory.upfrontDiff.text : $ctrl.configurationTile.upgrades.memory.diff.text,
                    price: $ctrl.configurationTile.isUpfront ? $ctrl.configurationTile.upgrades.memory.upfrontTotal.text : $ctrl.configurationTile.upgrades.memory.total.text
                }"
```

Reply:
Yes we can